pkgs/applications/virtualization/quickgui/default.nix

```diff
@@ -1,67 +1,60 @@
-{ stdenvNoCC
+{ fetchFromGitHub
+, makeDesktopItem
+, copyDesktopItems
 , lib
-, fetchurl
-, autoPatchelfHook
-, dpkg
-, wrapGAppsHook3
-, quickemu
+, flutter
 , gnome
+, quickemu
 }:
-
-stdenvNoCC.mkDerivation rec {
+let
+  runtimeBinDependencies = [ gnome.zenity ];
+in
+flutter.buildFlutterApplication rec {
   pname = "quickgui";
-  version = "1.2.8";
-
-  src = fetchurl {
-    url = "https://github.com/quickemu-project/quickgui/releases/download/v${version}/quickgui_${version}-1_lunar1.0_amd64.deb";
-    sha256 = "sha256-crnV7OWH5UbkMM/TxTIOlXmvqBgjFmQG7RxameMOjH0=";
+  version = "1.2.9";
+  src = fetchFromGitHub {
+    owner = "quickemu-project";
+    repo = "quickgui";
+    rev = version;
+    hash = "sha256-B/vhrL4thzSnWmPGzEgwphYwomj4a9EP/EqiULK5hsk=";
   };
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
 
-  nativeBuildInputs = [
-    autoPatchelfHook
-    dpkg
-    wrapGAppsHook3
-  ];
-
-  buildInputs = [
-    quickemu
-    gnome.zenity
-  ];
-
-  strictDeps = true;
-
-  unpackCmd = "dpkg-deb -x $curSrc source";
-
-  installPhase = ''
-    runHook preInstall
-
-    mv usr $out
+  gitHashes = {
+    window_size = "sha256-XelNtp7tpZ91QCEcvewVphNUtgQX7xrp5QP0oFo6DgM=";
+  };
 
-    runHook postInstall
-  '';
+  # These things are added to LD_LIBRARY_PATH, but not PATH
+  runtimeDependencies = [ quickemu ];
```

Comment:
I don't think quickemu needs to be included here, it builds and works without it being here when it's already included in runtimeBinDependencies.

Reply:
Rebased and made the requested change to include `quickemu` in the `runtimeBinDependencies`.